### ISSUE TITLE
cherrytree.desktop

### ISSUE THREAD

**Issue by adminka-root** [giuspen/cherrytree#2077]
It seems to me more convenient to launch the application with command:
`Exec=cherrytree --new_window %f`
instead of:
`Exec=cherrytree %f`

This allows you to launch multiple instances of the application from the menu, which is sometimes convenient (for example, when copying and pasting individual elements from one window to another).

If possible, set the suggested Exec by default, please :)

**Comment by giuspen** [comment]
I was skeptic but after checking what other apps are doing turns out the standard is what you asked for, so it's done.

**Comment by Breizhux** [comment]
I came to make the same suggestion, but I see that it has already been proposed... Thanks for the change!
I would like to bring another solution, which could allow to choose easily between "new instance" / "display the current instance".
In my case, I added these few lines at the end of the cherrytree.desktop file. To open a new instance I just have to right click on the launcher, then click on "New instance".
```
Actions=New-Instance;
[Desktop Action New-Instance]
Name=New Instance
Name[fr]=Nouvelle instance
Exec=cherrytree -N
```

The advantage is to keep both possibilities at your fingertips ;)
